torch/csrc/jit/python_ir.cpp

```diff
@@ -485,21 +485,9 @@ void initPythonIRBindings(PyObject * module_) {
   .def_readonly("user",&Use::user)
   .def_readonly("offset",&Use::offset);
 
-  m.def("_jit_import_graph", [](const std::string& serialized_graph) {
-    std::vector<at::Tensor> initializers;
-    auto graph = ImportIRGraph(serialized_graph, initializers);
-    std::vector<torch::autograd::Variable> variables;
-    variables.reserve(initializers.size());
-    for (auto& tensor : initializers) {
-      variables.push_back(torch::autograd::make_variable(
-          std::move(tensor), /*requires_grad=*/false));
-    }
-    return std::make_tuple(graph, variables);
-  });
   m.def("_jit_import_module", [](const std::shared_ptr<script::Module> module,
-                                 const std::string& serialized_module,
-                                 const std::unordered_map<std::string, std::string>& storages) {
-    ImportIRModule(module, serialized_module, storages);
+                                 const std::string& filename) {
+    ImportIRModule(module, filename);
```

Comment:
Let's make a C++ API as well:
```
   module = torch::jit::load(filename)
```

The function `_jit_import_module` should be added a method to ScriptModule's python bindings instead, `m._load(filename)`.